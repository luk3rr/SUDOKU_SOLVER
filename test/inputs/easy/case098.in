087002005 000600100 600870000 710300000 400086050 802700000 001020368 000040010 000060900 
