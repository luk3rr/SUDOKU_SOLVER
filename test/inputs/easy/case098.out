Solving the following grid:
0 8 7 | 0 0 2 | 0 0 5 
0 0 0 | 6 0 0 | 1 0 0 
6 0 0 | 8 7 0 | 0 0 0 
------+-------+------
7 1 0 | 3 0 0 | 0 0 0 
4 0 0 | 0 8 6 | 0 5 0 
8 0 2 | 7 0 0 | 0 0 0 
------+-------+------
0 0 1 | 0 2 0 | 3 6 8 
0 0 0 | 0 4 0 | 0 1 0 
0 0 0 | 0 6 0 | 9 0 0 


Solution found :')

1 8 7 | 4 3 2 | 6 9 5 
2 3 4 | 6 5 9 | 1 8 7 
6 5 9 | 8 7 1 | 2 4 3 
------+-------+------
7 1 5 | 3 9 4 | 8 2 6 
4 9 3 | 2 8 6 | 7 5 1 
8 6 2 | 7 1 5 | 4 3 9 
------+-------+------
9 4 1 | 5 2 7 | 3 6 8 
3 7 6 | 9 4 8 | 5 1 2 
5 2 8 | 1 6 3 | 9 7 4 

Algorithm: BFS
Total time: 23 ms
Total expanded states: 3786
